biodatacatalyst-ui/src/main/webapp/picsureui/header/header.hbs

```diff
@@ -108,6 +108,7 @@
                 {{#if jupyterExampleLink}}<li id="jupyter-examples-option" class="dropdown" aria-label="Jupyter Examples, clicking here will open a link  in a new tab"><a href="{{{jupyterExampleLink}}}">Jupyter Examples</a></li>{{/if}}
                 {{#if documentationLink}}<li id="documentation-option" class="dropdown" aria-label="BioData Catalyst Documentation, clicking here will open a link  in a new tab"><a href="{{{documentationLink}}}">BioData Catalyst Documentation</a></li>{{/if}}
                 {{#if helpLink}}<li id="contact-us-option" class="dropdown" aria-label="Contact Us link, clicking here will open a link in a new tab"><a href="{{{helpLink}}}">Contact Us</a></li>{{/if}}
+                <li id="terms-of-use-option" class="dropdown" aria-label="Terms of use, clicking here will open a modal"><a>Terms of Service</a></li>
```

Comment:
Should the aria label also say Terms of Service or should the link text say Terms of use? These should probably match.

Reply:
Good catch, its supposed to be Terms of Use